pom.xml

```diff
@@ -14,20 +14,39 @@
   </licenses>
 
   <properties>
-    <aspectj.version>1.9.19</aspectj.version>
-    <raml-module-builder.version>35.0.1</raml-module-builder.version>
-    <vertx.version>4.3.4</vertx.version>
-    <junit.jupiter.version>5.8.2</junit.jupiter.version>
-    <rest-assured.version>5.1.1</rest-assured.version>
-    <log4j-slf4j-impl.version>2.19.0</log4j-slf4j-impl.version>
-    <folio-di-support.version>1.6.0</folio-di-support.version>
-    <wiremock.version>2.26.3</wiremock.version>
-    <jsonschema_paths>acq-models/mod-orgs/schemas,raml-util/schemas,acq-models/common/schemas,acq-models/acquisitions-unit/schemas</jsonschema_paths>
+    <!--Project properties-->
+    <java.version>17</java.version>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <generated_sources_dir>${project.build.directory}/generated-sources</generated_sources_dir>
     <jsonschema2pojo_output_dir>${generated_sources_dir}/jsonschema2pojo</jsonschema2pojo_output_dir>
-    <log4j.version>2.17.2</log4j.version>
+    <jsonschema_paths>acq-models/mod-orgs/schemas,raml-util/schemas,acq-models/common/schemas,acq-models/acquisitions-unit/schemas</jsonschema_paths>
+
+    <!--Dependency Management Properties-->
+    <log4j.version>2.20.0</log4j.version>
+
+    <!--Dependency properties-->
+    <aspectj.version>1.9.20.1</aspectj.version>
+    <raml-module-builder.version>35.1.0</raml-module-builder.version>
```

Comment:
seems we could not use 35.1.0 based on statement in this ticket https://issues.folio.org/browse/RMB-983: "RMB versions before 35.1.0 are not patched" and so we did not make any pathces in Poppy.
Now we are going to increase and 35.1.0 that contains some vulnerability, so we need to increase to 35.1.1 and Vertx 4.4.6

Reply:
Thanks for review. Done